native-hello-module/README.md

```diff
@@ -66,23 +50,34 @@ The following modularized Java Hello World example
         >     exports hello;
         > }
 
-can be executed via Java using the following command:
-    
-    $ java --module-path target/HelloModule-1.0-SNAPSHOT.jar --module HelloModule
-    Hello from Java Module: HelloModule
-    
-With GraalVM you can **now build Java modules** into native executables, using the following syntax:
+1. Compile and package the project with Maven:
+    ```bash
+    $JAVA_HOME/bin/mvn package
+    ```
+    This creates a JAR file with all dependencies embedded: `target/HelloModule-1.0-SNAPSHOT.jar`
 
-    $ native-image --module-path target/HelloModule-1.0-SNAPSHOT.jar --module HelloModule
+2. Test running it on GraalVM’s JDK:
+    ```bush    
+    $JAVA_HOME/bin/java --module-path target/HelloModule-1.0-SNAPSHOT.jar --module HelloModule
+    ```
+    You should see the next  output:
+    ```
+    Hello from Java Module: HelloModule
+    ```
 
-    [hellomodule:10847]    classlist:     513.59 ms,  0.96 GB
-    [hellomodule:10847]        (cap):     386.65 ms,  0.96 GB
-    ....
-    [hellomodule:10847]        image:     968.93 ms,  4.85 GB
-    [hellomodule:10847]        write:     201.81 ms,  4.85 GB
-    [hellomodule:10847]      [total]:  19,216.19 ms,  4.85 GB
+3. Build this module into a native executable:
+    ```bush 
+    $JAVA_HOME/bin/native-image --module-path target/HelloModule-1.0-SNAPSHOT.jar --module HelloModule
+    ```
                                       
-It builds the modular Java app into a native executable that can be executed with:
-    
-    $ ./hellomodule 
+    It builds the modular Java application into a native executable called `hellomodule` in the project root directory that you can execute:
```

Comment:
Change to:

It builds the modular Java application into a native executable called `hellomodule` in the project root. Run it: